### PR TITLE
fix(demo): make `Stackblitz` to be compatible with `Taiga UI 3.0`

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [X] Build or CI related changes
- [X] Documentation content changes

## What is the current behavior?
Unfortunately, previous [stackblitz-hack](https://github.com/Tinkoff/taiga-ui/blob/main/projects/demo/src/modules/app/stackblitz/project-files/src/app/app.module.ts.md) does not works:
```ts
function tuiPropagateModules<T>(entryPoint: T) {
  const modules = [];

  for (const name in entryPoint) {
    if (name.endsWith('Module')) {
      modules.push(entryPoint[name]);
    }
  }

  return modules;
}
```

It causes error:
```console
Error in src/app/app.module.ts (19:17)
Value at position 1 in the NgModule.declarations of AppModule is not a reference
Value could not be determined statically.
```

See this issue https://github.com/angular/angular/issues/42550

## What is the new behavior?
New hack :)

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo/src/modules/app/stackblitz/utils.ts [hunk 1]
```diff
@@ -67,3 +67,95 @@ export const getSupportModules = (
...
+     * `Warning: Critical dependency: the request of a dependency is an expression`
+     * */
+    const cdk = getAllModules(await import(`@taiga-ui/cdk`));
+    const core = getAllModules(await import(`@taiga-ui/core`));
+    const kit = getAllModules(await import(`@taiga-ui/kit`));
+    const charts = getAllModules(await import(`@taiga-ui/addon-charts`));
+    const commerce = getAllModules(await import(`@taiga-ui/addon-commerce`));
+    const editor = getAllModules(await import(`@taiga-ui/addon-editor`));
+    const mobile = getAllModules(await import(`@taiga-ui/addon-mobile`));
+    const table = getAllModules(await import(`@taiga-ui/addon-table`));
```
Comment (splincode) [inline projects/demo/src/modules/app/stackblitz/utils.ts:102]: why not Promise.all?

File: projects/demo/src/modules/app/stackblitz/utils.ts [hunk 2]
```diff
@@ -67,3 +67,95 @@ export const getSupportModules = (
...
+import {PolymorpheusModule} from '@tinkoff/ng-polymorpheus';
+import {RouterModule} from '@angular/router';
+
+export const ALL_TAIGA_UI_MODULES = [
+    BrowserModule,
+    BrowserAnimationsModule,
+    FormsModule,
+    ReactiveFormsModule,
+    PolymorpheusModule,
+    RouterModule.forRoot([]),
```
Comment (splincode) [inline projects/demo/src/modules/app/stackblitz/utils.ts:142]: why not

```ts
@NgModule({
  imports: [
    BrowserModule,
    BrowserAnimationsModule,
    FormsModule,
    ReactiveFormsModule,
    PolymorpheusModule,
    RouterModule.forRoot([]),

  /**
   * Don't use this approach,
   * it's a workaround for stackblitz
   */
    ...ALL_TAIGA_UI_MODULES
  ],
```

Comment (splincode) [reply projects/demo/src/modules/app/stackblitz/utils.ts:142]: Oh, I see https://github.com/angular/angular/issues/42550

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/app/stackblitz/utils.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/app/stackblitz/utils.ts:102]: My mistake, we need `Promise.all` here, thanks)

Comment (nsbarsukov) [reply projects/demo/src/modules/app/stackblitz/utils.ts]: fixed ✅

**Review by vladimirpotekhin — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**